### PR TITLE
Update and apply data checkers

### PR DESCRIPTION
**Pull request by sparshg**
I have constrained gnuchess version < 6.0.0 because of weird bugs if we update that.. like the pieces don't appear at all in 6.2.9 and black doesn't move its turn in 6.0.0 even though all pieces appear.

### CODE REVIEW

File: chess-fix-clip.patch [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
++        clipboard = Gtk.Clipboard.get(Gdk.SELECTION_CLIPBOARD)
++        clipboard.set_text(self.tag_pairs() + self._gnuchess.copy_game(), -1)
+ 
+     def _paste_cb(self, *args):
+         ''' Pasting '''
+-        clipboard = Gtk.Clipboard()
++        clipboard = Gtk.Clipboard.get(Gdk.SELECTION_CLIPBOARD)
+         move_list = self._parse_move_list(clipboard.wait_for_text())
+         if move_list is not None:
+             self._gnuchess.restore_game(move_list)
```
Comment (tchx84) [inline chess-fix-clip.patch:20]: @sparshg as a follow up, can you please upstream this fix?

File: org.sugarlabs.Chess.json [hunk 2]
```diff
@@ -32,7 +32,14 @@
                 {
                     "type": "archive",
                     "url": "http://ftp.gnu.org/gnu/chess/gnuchess-5.08.tar.gz",
-                    "sha256": "c4e49e0dec210f5d131a02ff89588b482787cd702a08456791ba9100b4c6ffc0"
+                    "sha256": "c4e49e0dec210f5d131a02ff89588b482787cd702a08456791ba9100b4c6ffc0",
+                    "x-checker-data": {
+                        "type": "html",
+                        "url": "http://ftp.gnu.org/gnu/chess/",
+                        "versions": {"<": "6.0.0"},
```
Comment (tchx84) [inline org.sugarlabs.Chess.json:39]: @sparshg as a follow up, can you create an upstream issue?

File: chess-fix-clip.patch (hunk 1 above)
Comment (sparshg) [reply chess-fix-clip.patch:20]: This fix is the latest commit in the upstream but the version is released just before this commit. (Thats what I asked in #3). With the next release we can delete `chess-fix-clip.patch`

File: org.sugarlabs.Chess.json (hunk 2 above)
Comment (sparshg) [reply org.sugarlabs.Chess.json:39]: Created an issue, https://github.com/sugarlabs/sugarchess/issues/18

**Pull request merged**